src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalAmount = days * 40;
```

Comment:
The daily rate of '40' is a 'magic number' (a hard-coded value with unexplained meaning). For better maintainability, consider declaring it as a constant at the top of your function.